yt/fields/xray_emission_fields.py

```diff
@@ -204,11 +204,15 @@ def add_xray_emissivity_field(ds, e_min, e_max, redshift=0.0,
                                "Perhaps you should specify a constant metallicity instead?")
 
     if table_type == "cloudy":
-        # Cloudy wants to know the total number density of hydrogen
-        H_field = "H_nuclei_density"
+        # Cloudy wants to scale by nH**2
```

Comment:
Could we have the second field be a string argument, and just have one function def?